lychee-lib/src/types/input.rs

```diff
@@ -140,11 +140,7 @@ impl Input {
                 Ok(url) if url.scheme() == "http" || url.scheme() == "https" => {
                     InputSource::RemoteUrl(Box::new(url))
                 }
-                Ok(_) => {
-                    // URL parsed successfully, but it's not http or https
-                    return Err(ErrorKind::InvalidFile(PathBuf::from(value)));
-                }
-                _ => {
+                Ok(_) | _ => {
```

Comment:
Another change only because I'm running on Windows.

If either of the Windows specific changes are problematic I can pull them out, I'll be running it via the docker image anyways.

Reply:
...and this one did cause a test failure, so just reverted it